### DISCUSSION TITLE
Is this a bug? (Breaking changes after update)

### DISCUSSION THREAD

**Discussion by OxNinja** [NvChad/NvChad#1197] [Q&A] [answered]
After updating NVCHad `<leader> + uu` I got the following errors:

![image](https://user-images.githubusercontent.com/57400445/173223855-49673741-710d-488c-9c06-abc56b72e1e1.png)

Then the lua plugin loader/updater was blank, I had to kill vim.

To get the actual update I uninstalled nvchad and reinstalled it. I wonder if this is a bug? It happened on 2 different devices.

**Comment by siduck** [comment, marked as answer]
@OxNinja do a packersync again
